### PR TITLE
Added CONTRIBUTING.md with Contribution guidelines

### LINKED ISSUE #5: Contribution guidelines missing
Being an open-source project, it would be nice to have a [CONTRIBUTING.md](https://en.wikipedia.org/wiki/Contributing_guidelines).

This would give way to a lot more contributions and easy entry into the project for new contributors.

### PR DESCRIPTION
**Pull request by pankaj892**
Closes #5

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# 🙌 Contribution Guidelines
+
+**First of all thanks for taking time and interest in the project 👍**
+
+## First time Contributors
+If you're new to open source and don't know how to contribute to open source projects then check out this [link](https://github.com/firstcontributions/first-contributions)
```
Comment (Vibhu-Agarwal) [inline CONTRIBUTING.md]:
```suggestion
If you're new to open source and don't know how to contribute to open-source projects then check out this [guide-project](https://github.com/firstcontributions/first-contributions) 
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+## Push changes
+- Push changes on your local machine to GitHub using command 
+```
+git push origin master
+```
+
+## Compare and Pull Request
+- Compare the branches and start a Pull Request to merge the changes in the master branch
+
+## You contributed to the project Congrats on your contribution 🎉
```
Comment (Vibhu-Agarwal) [inline CONTRIBUTING.md]:
```suggestion
## You contributed to the project. Congrats on your contribution 🎉
```

**Review by Vibhu-Agarwal — CHANGES_REQUESTED** [review]
@pankaj892 The PR looks great. Just fix a few typos pointed out in the comments.
Also, you can add a few other things in the contributing guidelines by taking some inspiration from [the link mentioned in the issue](https://github.com/Vibhu-Agarwal/potter_spells/issues/5#issuecomment-743755142) this PR fixes.

**Comment by Vibhu-Agarwal** [comment]
@pankaj892 I would like to point out a few things though.
- You were assigned issue #1, but you started working on issue #5.
  - No updates on #1 yet; picked up another issue (#5).
  - Assigned issues should **always** be given higher priority to work on.
  - Started working on #5 without even getting assigned for it. Didn't even volunteer to work on #5 by mentioning it in the comments. This really hampers the work-flow of contributors in open-source projects.
- It's ironic that you made the PR from your `master` branch because you yourself added the instructions in the CONTRIBUTING.md to use feature-branches to work on issues.

**Comment by pankaj892** [comment]
Updated CONTRIBUTING.md

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+## Push changes
+- Push changes on your local machine to GitHub using command 
+```
+git push origin master
+```
+
+## Compare and Pull Request
+- Compare the branches and start a Pull Request to merge the changes in the master branch
+
+## You contributed to the project. Congrats on your contribution 🎉 and welcome to Open Source
```
Comment (Vibhu-Agarwal) [inline CONTRIBUTING.md]: Line too long for a section heading

**Review by Vibhu-Agarwal — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 3 above)
Comment (pankaj892) [reply CONTRIBUTING.md]: Fixed section heading check now @Vibhu-Agarwal

**Pull request merged**